exercises/resistor-color/resistor_color.go

```diff
@@ -0,0 +1,24 @@
+// This is a "stub" file.  It's a little start on your solution.
```

Comment:
I'd remove all the stub comments. This is a later exercise, so students know by now where to put code.